src/evict/evict_page.c

```diff
@@ -876,6 +883,9 @@ __evict_reconcile(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t evict_flags)
     if (ret != 0)
         WT_STAT_CONN_INCR(session, cache_eviction_fail_in_reconciliation);
 
+    if (is_application_thread_snapshot_refreshed)
```

Comment:
Please combine this if cond and next one as if {...} else if{...} similar to in lines 849 and 857

Reply:
Fixed in 94e56fbdda9d6608262f541e53ab0b05a363113f